docs/Common/Enums/index.md

```diff
@@ -865,30 +865,32 @@
 |VEHICLEDATA_FUELCONSUMPTION|5||
 |VEHICLEDATA_EXTERNTEMP|6||
 |VEHICLEDATA_VIN|7||
-|VEHICLEDATA_PRNDL|8||
-|VEHICLEDATA_TIREPRESSURE|9||
-|VEHICLEDATA_ODOMETER|10||
-|VEHICLEDATA_BELTSTATUS|11||
-|VEHICLEDATA_BODYINFO|12||
-|VEHICLEDATA_DEVICESTATUS|13||
-|VEHICLEDATA_ECALLINFO|14||
-|VEHICLEDATA_AIRBAGSTATUS|15||
-|VEHICLEDATA_EMERGENCYEVENT|16||
-|VEHICLEDATA_CLUSTERMODESTATUS|17||
-|VEHICLEDATA_MYKEY|18||
-|VEHICLEDATA_BRAKING|19||
-|VEHICLEDATA_WIPERSTATUS|20||
-|VEHICLEDATA_HEADLAMPSTATUS|21||
-|VEHICLEDATA_BATTVOLTAGE|22||
-|VEHICLEDATA_ENGINETORQUE|23||
-|VEHICLEDATA_ACCPEDAL|24||
-|VEHICLEDATA_STEERINGWHEEL|25||
-|VEHICLEDATA_TURNSIGNAL|26||
-|VEHICLEDATA_FUELRANGE|27||
-|VEHICLEDATA_ENGINEOILLIFE|28||
-|VEHICLEDATA_ELECTRONICPARKBRAKESTATUS|29||
-|VEHICLEDATA_CLOUDAPPVEHICLEID|30| Parameter used by cloud apps or the policy server to identify a head unit|
-|VEHICLEDATA_OEM_CUSTOM_DATA|31||
+|VEHICLEDATA_GEARSTATUS|8||
+|VEHICLEDATA_PRNDL|9||
+|VEHICLEDATA_TIREPRESSURE|10||
+|VEHICLEDATA_ODOMETER|11||
+|VEHICLEDATA_BELTSTATUS|12||
+|VEHICLEDATA_BODYINFO|13||
+|VEHICLEDATA_DEVICESTATUS|14||
+|VEHICLEDATA_ECALLINFO|15||
+|VEHICLEDATA_AIRBAGSTATUS|16||
+|VEHICLEDATA_EMERGENCYEVENT|17||
+|VEHICLEDATA_CLUSTERMODESTATUS|18||
+|VEHICLEDATA_MYKEY|19||
+|VEHICLEDATA_BRAKING|20||
+|VEHICLEDATA_WIPERSTATUS|21||
+|VEHICLEDATA_HEADLAMPSTATUS|22||
+|VEHICLEDATA_BATTVOLTAGE|23||
+|VEHICLEDATA_ENGINETORQUE|24||
+|VEHICLEDATA_ACCPEDAL|25||
+|VEHICLEDATA_STEERINGWHEEL|26||
+|VEHICLEDATA_TURNSIGNAL|27||
+|VEHICLEDATA_FUELRANGE|28||
+|VEHICLEDATA_ENGINEOILLIFE|29||
+|VEHICLEDATA_ELECTRONICPARKBRAKESTATUS|30||
+|VEHICLEDATA_CLOUDAPPVEHICLEID|31| Parameter used by cloud apps or the policy server to identify a head unit|
+|VEHICLEDATA_OEM_CUSTOM_DATA|32||
+
```

Comment:
```suggestion
|VEHICLEDATA_OEM_CUSTOM_DATA|32||
```
Just to follow the convention for other sections

Reply:
fixed in 78995a7a3f364d0a93ec57cb1b5b59c0099dffb6